code/modules/mob/living/bot/medibot.dm

```diff
@@ -23,19 +31,57 @@
 	var/treatment_emag =  /decl/material/liquid/venom
 	var/declare_treatment = 0 //When attempting to treat a patient, should it notify everyone wearing medhuds?
 
+	// Are we tipped over?
+	var/is_tipped = FALSE
+	//How panicked we are about being tipped over (why would you do this?)
+	var/tipped_status = MEDBOT_PANIC_NONE
+	//The name we got when we were tipped
+	var/tipper_name
+	//The last time we were tipped/righted and said a voice line, to avoid spam
+	var/last_tipping_action_voice = 0
+
+/mob/living/bot/medbot/examine(mob/user)
+	. = ..()
+	if(tipped_status == MEDBOT_PANIC_NONE)
+		return
+
+	switch(tipped_status)
+		if(MEDBOT_PANIC_NONE to MEDBOT_PANIC_LOW)
+			to_chat(user, "It appears to be tipped over, and is quietly waiting for someone to set it right.")
+		if(MEDBOT_PANIC_LOW to MEDBOT_PANIC_MED)
+			to_chat(user, "It is tipped over and requesting help.")
+		if(MEDBOT_PANIC_MED to MEDBOT_PANIC_HIGH)
+			to_chat(user, SPAN_WARNING("They are tipped over and appear visibly distressed.")) // now we humanize the medbot as a they, not an it
+		if(MEDBOT_PANIC_HIGH to MEDBOT_PANIC_FUCK)
+			to_chat(user, SPAN_WARNING("They are tipped over and visibly panicking!"))
+		if(MEDBOT_PANIC_FUCK to INFINITY)
+			to_chat(user, SPAN_DANGER("They are freaking out from being tipped over!"))
+
 /mob/living/bot/medbot/handleIdle()
 	if(vocal && prob(1))
-		var/message = pick("Radar, put a mask on!", "There's always a catch, and it's the best there is.", "I knew it, I should've been a plastic surgeon.", "What kind of infirmary is this? Everyone's dropping like dead flies.", "Delicious!")
+		var/message_options = list(
```

Comment:
should probably be /static/

Reply:
same with other message lists

Reply:
some can't be statics because they have variables in them, but i've converted the ones that can be (i think)

Reply:
Probably clearer to move the static var declarations onto the type instead of doing it at local scope.